src/styles/components/_form.scss

```diff
@@ -26,7 +26,8 @@ $dso-fieldset-separator-color: $grijs-20;
 $dso-fieldset-separator-unit: $u4;
 
 .form,
-form {
+form,
+.dso-section-body {
```

Comment:
Waarom is dit nodig? al deze onderdelen staan toch nog steeds in het `form`? Toevallig zit er een `.dso-section-body` tussen, maar wat boeit dat?

Reply:
Omdat de `<form>` tag nu niet meer IN maar buiden de accordion staat. En je wilt hier styling op de `> fieldset > legend`.

Reply:
Ok